isala.local/app/models/HomeModel.php

```diff
@@ -1,15 +1,26 @@
 <?php
 
+require_once('../app/database/connection.php');
+require_once('../app/ldap/connection.php');
+
 class HomeModel
 {
     private $title;
+    private $ldap;
+
     public function __construct()
     {
         $this->title = 'Home';
+        $this->ldap = new LDAPConnection();
```

Comment:
Waar heeft Home een LDAP connectie voor nodig?

Reply:
Voor het controleren of de gebruiker geauthoriseerd is om een button te zien.